tools.js

```diff
@@ -0,0 +1,45 @@
+const normalizeVersion = require('./normalize-version');
+const { execSync } = require("child_process");
+
+module.exports = {
+  node: {
+    getVersion: "node --version",
+    getInstallInstructions(v) {
+     try {
```

Comment:
Nit: Might be worth putting this logic in a `hasNvm()` helper function.

Reply:
Sure, better readability is always good 🙂